Change the label and description of the PayPal Mark setting

rakmob42 opened the pull request:
<!-- Reference the source of this Pull Request. -->
<!-- Remove any which are not applicable. -->

### Description
<!-- Describe the changes made in this Pull Request and the reason for these changes. -->
This fixes https://github.com/woocommerce/woocommerce-gateway-paypal-express-checkout/issues/675. This is a simple change to the label and description of the "PayPal Mark" setting.

Previous:

> Label: Enable the PayPal Mark on regular checkout.
> Description: This enables the PayPal mark, which can be shown on regular WooCommerce checkout to use PayPal Checkout like a regular WooCommerce gateway.

New:

> Label: Show PayPal Checkout payment option on the regular WooCommerce checkout page.
> Description: Disabling this will hide PayPal mark from the regular WooCommerce checkout page.

I'm not sure whether I should call it `PayPal payment option` or `PayPal mark`. I think the `PayPal payment option` is much simpler to understand.



### Documentation
<!-- Will this change require new documentation or changes to existing documentation? -->
<!-- A good way to answer it is to ask: will more than one customer ever need to know about this? -->
- [x] This PR needs documentation (has the "Documentation" label).
<!-- For an extra 💯 include further details about which change requires documentation -->
Currently, there's no mention of "PayPal Mark" in our documentation. We can include it here: https://docs.woocommerce.com/document/paypal-express-checkout

Closes #675

Commit:
Change the description of the PayPal Mark setting

Currently the description isn't clear. The new one clearly communicates that disabling this option will hide PayPal option from the "regular WooCommerce checkout page".

james-allan commented:
Great first PR @codestor4 💯 

I agree that the previous language was pretty confusing. After looking at all the settings together though, I think we should use this opportunity to make the setting names more consistent. 

<img width="510.6" alt="Screen Shot 2020-06-17 at 3 54 34 pm" src="https://user-images.githubusercontent.com/8490476/84860743-f3d96780-b0b2-11ea-805d-d3950682a6af.png">

link: https://cloudup.com/cYoyaKbf0QR

All of the highlighted settings enable the PayPal buttons on the respective pages and they are all worded differently.

In my opinion using the simplest wording here is probably be the best approach. `Checkout on the x page` and `Enable PayPal Checkout on the x page` is concise and clear. The only issue with this is when it comes to the checkout settings. `Checkout on the checkout page` 😕 

What do you think about this **Enable on the Checkout Page** and **Enable PayPal Checkout on the regular checkout page.**

That would look like this:

<img width="490.5" alt="Screen Shot 2020-06-17 at 4 27 14 pm" src="https://user-images.githubusercontent.com/8490476/84862962-7401cc00-b0b7-11ea-858d-3d4615f5df90.png">

https://cloudup.com/cpghvgKhKsG

While we're here we should update the tooltip text too. For example https://cloudup.com/c_VUHFanwnB vs https://cloudup.com/cIrwakzLNz7 vs https://cloudup.com/cT6IgwnhjJ4

codestor4 commented:
Thanks for sharing these inputs, James. Based on that, here's what I propose:

#### Title
- Enable on the cart page
- Enable on the single product page
- Enable on the checkout page

#### Label
- Enable PayPal checkout button on the cart page
- Enable PayPal checkout button on the single product page
- Enable PayPal checkout button on the regular checkout page

#### Description (tool-tip)
I think we should remove tool-tips for all three of these because the title and label itself is getting repetitive and tool-tip will make us repeat these lines the third time.

@james-allan what do you think?

james-allan commented:
> what do you think?

Sounds good to me :) 

Just 2 little tweaks to the description text:

**_Enable PayPal Checkout buttons on the cart page_**

I've uppercased Checkout and pluralised buttons.

Commit:
Add consistency in title + label and remove desc

Adds consistency and clarity in the title and label fields and removes the description field.

james-allan commented:
Finally getting around to review this again. The changes I requested in my last comment were addressed in 518225a. This LGTM.
